newsletters/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+
+class department(models.Model):
+    '''
+    Stores the detail of department linked with its newsletters
+    '''
+    department = models.CharField(max_length=50,
+                                  blank=False,
+                                  null=False)
+
+    def __unicode__(self):
+        return "%s" % (self.department)
+
+
+class newsletters(models.Model):
+    '''
+    Stores the pdfs of newsletters
+    '''
+    title = models.CharField(null=False,
+                             max_length=25)
+    uploaded_by = models.ForeignKey(User)
+    year = models.DateField()
+    paper = models.FileField(upload_to="newsletters",
+                             blank=False,
+                             null=False)
+    department = models.ForeignKey(department)
+    created_at = models.DateTimeField()
+    modified_at = models.DateTimeField()
+
+    def create_newsletter(self):
```

Comment:
Can you please tell me the use case of this method?

Reply:
Sir, I had in mind of writing some test cases for this model but did not frame them. Will remove it for now and add this in a separate PR with test cases.